.gitmodules

```
@@ -1,3 +1,6 @@
 [submodule "github.com/libbpf/libbpf"]
 	path = 3rdparty/libbpf
 	url = https://github.com/libbpf/libbpf.git
+[submodule "testdata"]
```

nit: could we name it like the other submodule/Go modules, please?
```suggestion
[submodule "github.com/parca-dev/testdata"]
```

---

Good shout, thanks!